tests/integration/renderers/test_pydsl.py

```diff
@@ -23,26 +23,40 @@ def test_rendering_includes(self):
         inability to load custom modules inside the pydsl renderers. This
         is a FIXME.
         '''
-        try:
-            self.run_function('state.sls', ['pydsl.aaa'])
-
-            expected = textwrap.dedent('''\
-                X1
-                X2
-                X3
-                Y1 extended
-                Y2 extended
-                Y3
-                hello red 1
-                hello green 2
-                hello blue 3
-                ''')
-
-            with salt.utils.fopen('/tmp/output', 'r') as f:
-                self.assertEqual(sorted(f.read()), sorted(expected))
-
-        finally:
-            os.remove('/tmp/output')
+        self.run_function('state.sls', ['pydsl.aaa'])
+
+        expected = textwrap.dedent('''\
+            X1
+            X2
+            X3
+            Y1 extended
+            Y2 extended
+            Y3
+            hello red 1
+            hello green 2
+            hello blue 3
+            ''')
+
+        # Windows adds `linefeed` in addition to `newline`. There's also an
+        # unexplainable space before the `linefeed`...
+        if salt.utils.is_windows():
+            expected = 'X1 \r\n' \
+                       'X2 \r\n' \
+                       'X3 \r\n' \
+                       'Y1 extended \r\n' \
+                       'Y2 extended \r\n' \
+                       'Y3 \r\n' \
+                       'hello red 1 \r\n' \
+                       'hello green 2 \r\n' \
+                       'hello blue 3 \r\n'
+
+        with salt.utils.fopen('/tmp/output', 'r') as f:
+            ret = f.read()
+
+        os.remove('/tmp/output')
```

Comment:
The path is created by the test suite and some files are placed in that directory.